### PR TITLE
Update runtime

### PR DESCRIPTION
**Pull request by dvadgama**

### CODE REVIEW

File: .devcontainer/on-create.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+function install_packages(){
+    # install_packages
+    #  -install bash-completion required to use poertry bash-completion
+    apt-get update -y && apt-get install -y  bash-completion
+
+    echo 'source /etc/profile' >> ${HOME}/.bashrc
+}
+
+function install_poerty(){
```
Comment (shehzadashiq) [inline .devcontainer/on-create.sh:12]: Perhaps this should be named install_poetry()

File: .devcontainer/on-create.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        poetry self add poetry-plugin-up
+
+    fi
+}
+
+function main(){
+    # main
+    #  - execute functions in a given order
+    install_packages
+    install_poerty
```
Comment (shehzadashiq) [inline .devcontainer/on-create.sh:82]: The call here should be to install_poetry

**Review by shehzadashiq — APPROVED** [review]
LGTM

File: .devcontainer/on-create.sh (hunk 1 above)
Comment (dvadgama) [reply .devcontainer/on-create.sh]: done

**Pull request merged**